src/App.jsx

```diff
@@ -54,6 +55,12 @@ function App() {
 	const [stakeAmount] = useDebounce(stakeInput, 500.0);
 	const [apiConnected, setApiConnected] = React.useState(false);
 	const [isLoaded, setIsLoaded] = React.useState(false);
+	let [validators, setValidators] = React.useState([{name: 'None', amount: 0, avatar: 'default', risk: 0.00}]);
+	const [data, setData] = React.useState({
```

Comment:
Could we have a more descriptive name for this const?

Reply:
changed to suggValidatorsData